### DISCUSSION TITLE
Support for mobile

### DISCUSSION THREAD

**Discussion by kemilandmeta** [use-cardano/use-cardano#98] [Q&A] [answered]
Any option to connect to mobile wallet from the web application?

**Comment by GGAlanSmithee** [comment]
I haven't added any explicit support for mobile wallets, I need to do some testing for this. Maybe it just works, and if not, I need to research what the underlying abilities there are with Lucid.

**Reply by kemilandmeta** [reply]
I feel it's not that easy,

I checked spacebuds website from a mobile and I have both eternl and flint but it was not able to detect the wallets,

**Reply by GGAlanSmithee** [reply]
ok, I see. I will ask Alesandro if he has any pointers and come back to you

**Comment by GGAlanSmithee** [comment, marked as answer]
@kemilandmeta I realize I never got back to you - sorry!

I got an answer from the Lucid discord:

> Hello, yes you can connect via mobile with flint. You need [...] flint mobile browser

So, the answer is that you you need an app to enhance your mobile browser (just as a wallet browser plugin enhances your desktop browser). For Cardano, this is done with the flint mobile wallet for now.

**Reply by GGAlanSmithee** [reply]
I guess gero and kraken also are mobile wallets

**Comment by kemilandmeta** [comment]
This is perfect tried flient and working perfectly will try both  gero and kraken as well, thanks for the support. I will close this discussion
